tests/common.sh

```diff
@@ -1,12 +1,14 @@
 #!/bin/sh
     
 L="$(pwd)/../src/snap-run"
+export L
 
 TMP="$(mktemp -d)"
-trap "rm -rf $TMP" EXIT
+trap 'rm -rf $TMP' EXIT
 
 export SNAPPY_LAUNCHER_SECCOMP_PROFILE_DIR="$TMP"
 export SNAPPY_LAUNCHER_INSIDE_TESTS="1"
+export UBUNTU_CORE_LAUNCHER_NO_ROOT=1
```

Comment:
This was supposed to be moved from run-tests so that each test was runnable standalone.